scripts/install-cni.sh

```diff
@@ -98,7 +98,7 @@ token=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
 host=${KUBERNETES_SERVICE_HOST}
 # If host constains a colon (:), it is an IPv6 address, hence needs wrapping
```

Comment:
"constains" is a typo.

Reply:
I fixed this myself from GitHub UI, but this PR needs squashing before merge.

Reply:
Thanks! Squashed